index.rst

```diff
@@ -48,7 +48,353 @@
+
+
+Schedule
+========
+
+We are planning toward the following objectives:
+
+AP operations
+-------------
+
+* First day running AP against Cassandra is attempted - mid-February 2024
+* Successfully running AP against Cassandra on AuxTel data - mid-March 2024
+* First day to attempt replicating Cassandra to Postgres - beginning of March 2024
+* Stable running of AP on Cassandra on AuxTel data with replication to Postgres - end of March 2024
+
+There is comfortable schedule margin available for these activities up through the start of May
+if there are unexpected problems.
+
+* Implementation of automated post-embargo sweeps to ``/repo/prompt`` - (TBD)
+* Implementation of 30-day cache purges - TBC if this will even be done in early operations
+  
+
+Data services
+-------------
+
+* Exposure of ``/repo/embargo`` images via image services - prototype currently running
+* Exposure of the expected user-facing image data products from ``/repo/prompt`` via image service - (TBD)
+* Exposure of the PPDB tables from Postgres via TAP - April 2024 or as soon thereafter as the actual data are available
+
+Is this the same Postgres server and the same TAP service as the one serving ``/repo/embargo`` ObsCore?
+
+Notebook Aspect
+---------------
+
+* USDF RSP Nublado services are already in place
+* Some work may still be needed on configuring appropriate "short names" for Prompt Products data sources
+* Main effort will probably be to develop demonstration / tutorial notebooks to exercise
+  realistic science-user data access use cases on the released data
+
+Portal Aspect
+-------------
+
+* Basic Portal is available in USDF RSP now
+* Configuration tweaks are needed soon to shift focus from DP0.2/DC2 universe to the real universe
+* Refreshed Portal UI deployed on integration clusters by end of February, production by mid-April 2024
+* "Simplified query" screens aimed at PP use cases - April 2024
+  
+
+Data Sources
+============
+
+To avoid the risk of rapid obsolescence, we avoid mentioning the dates of the various upcoming stages of
+data acquisition and release here.
+These dates are more authoritatively addressed in RTN-011.
+
+AuxTel/LATISS
+-------------
+
+Initially the data available for use with "PP Preview" will be AuxTel/LATISS data taken on the Summit.
+
+While it would be possible to do an AP-like processing of a simulated Rubin dataset, or a collection of imaging data from an ``afw``-compatible project such as DECam, the main point of the "PP Preview" is to exercise and demonstrate the ability to serve *progressively* the results of ongoing processing, so a one-time static processing of an existing data collection would not meet this need.
+
+An observing run with AuxTel is currently done approximately every two weeks.
```

Comment:
Now approximately every week, with occasional long breaks for maintenance.

Reply:
Said differently, AuxTel generally runs on weeknights, again with breaks for maintenance.